mesonbuild/modules/rust.py

```diff
@@ -69,18 +73,25 @@ class FuncBindgen(TypedDict):
      KwargInfo('is_parallel', bool, default=False),
 ]
 
+def no_spaces_validator(arg: T.Optional[T.Union[str, T.List]]) -> T.Optional[str]:
+    if any((bool(re.search(r"\s", x)) for x in arg)):
```

Comment:
```suggestion
    if any((bool(re.search(r'\s', x)) for x in arg)):
```

Reply:
Also parens to produce a generator expression aren't needed inside a function argument like this.

Reply:
Although actually now that i think about it, why do you need regex at all here? Can't you use `any(' ' in x for x in arg)`? The difference here is whether you detect tab or newline characters, but this is about arguments written as kwargs in a meson.build file, right?

Reply:
Yes, it's the `args` kwarg. I need to forbid `rust.doctest(..., args: ['foo bar'])`. But the user can try to be fun and use tabs instead of spaces. :/